mypy/version.py

```diff
@@ -1 +1,11 @@
+import os
+from mypy import git
+
 __version__ = '0.4.6-dev'
+
+mypy_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+if git.is_git_repo(mypy_dir) and git.have_git():
```

Comment:
When run from a repo, these seem to add maybe 30 msec to the mypy runtime when there is a warm cache. This looks like up to 6% slowdown, which is not terrible if most users are using a pip-installed version. However, this could be much worse on cold cache. Can you check the effect on mypy runtime when run straight after restart?